mongodbatlas.go

```diff
@@ -110,63 +144,53 @@ func (m *MongoDBAtlas) CreateUser(ctx context.Context, statements dbplugin.State
 
 	_, _, err = client.DatabaseUsers.Create(ctx, m.ProjectID, databaseUserRequest)
 	if err != nil {
-		return "", "", err
+		return newdbplugin.NewUserResponse{}, err
 	}
-	return username, password, nil
-}
 
-// RenewUser is not supported on MongoDB, so this is a no-op.
-func (m *MongoDBAtlas) RenewUser(ctx context.Context, statements dbplugin.Statements, username string, expiration time.Time) error {
-	// NOOP
-	return nil
+	resp := newdbplugin.NewUserResponse{
+		Username: username,
+	}
+
+	return resp, nil
 }
 
-// RevokeUser drops the specified user from the authentication database. If none is provided
-// in the revocation statement, the default "admin" authentication database will be assumed.
-func (m *MongoDBAtlas) RevokeUser(ctx context.Context, statements dbplugin.Statements, username string) error {
+func (m *MongoDBAtlas) UpdateUser(ctx context.Context, req newdbplugin.UpdateUserRequest) (newdbplugin.UpdateUserResponse, error) {
+	if req.Password == nil {
```

Comment:
This will error when `UpdateUser` is called with just a change to the expiration which happens when renewing the secret. Instead, I recommend doing something like:
```
if req.Password != nil {
  // ...
  // change password
  // ...
}
// Explicitly comment that expirations are a no-op
```

Reply:
Addressed in 73812a4 (partially) and 5574448.